### ISSUE TITLE
How does this work with Apple/Google terms of condition?

### ISSUE THREAD

**Issue by elie222** [xcarpentier/react-native-stripe-api#11]
I thought all payments had to go through Apple/Google so they get their 30% cut (or whatever the number is)

**Comment by pmcguane** [comment]
incorrect, look at apps like eat now, grub hub, menu log etc.. even eBay

**Comment by elie222** [comment]
That didn't answer the question. 
So it seems that when you're selling physical items it's fine. For in app purchases you can't use stripe though.

**Comment by xcarpentier** [comment]
@elie222 thanks for this issue. Do you have a link about that ?

**Comment by Rob117** [comment]
@xcarpentier  @elie222  https://www.quora.com/What-is-Apples-policy-on-in-app-credit-card-payments

You can use in app purchases if the purchase in question is _not_ an iOS specific purchase, it seems
